files/scripts/lldp.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+. /usr/local/bin/asic_status.sh
```

Comment:
@abdosi to take a look as well - adding this new script /usr/local/bin/lldp.sh , which invokes /usr/bin/lldp.sh